include/ccl_power.h

```diff
@@ -64,6 +66,19 @@ double ccl_nonlin_matter_power(ccl_cosmology * cosmo, double k, double a,int * s
  * @return sigma(R).
  */
 double ccl_sigmaR(ccl_cosmology *cosmo, double R, int * status);
+double ccl_sigmaRz(ccl_cosmology *cosmo, double R, double a, int * status);
```

Comment:
I'd suggest to actually replace sigmaR with this function
It also seems like you're only making sigmaR (and sigmaV) transparent to python, but I would actually do it for the more general ones. That's why I would just replace sigmaR with a new sigmaR that also takes in an 'a' parameter. This could be made to default to a=1 at the python level. Does this make sense?

Reply:
Yes, let me try and do this.

The only concern I have relates to the approximation of scale-independent growth here. This somehow relates to my issue #344 . Clearly this is wrong in massive-neutrino cosmologies, but it is also slightly wrong all the time anyway because the baryons and CDM have different power spectra really. I guess that by using the approximation of scale-independent growth here you run the risk of someone calling ```sigmaR(z=1000)``` and getting some answer the meaning of which is debatable.

Reply:
Done

Reply:
How do I make sure that a=1 is the default with python? What files do I have to change to make this happen? Or is it going to be easier to change the sigmaR function so that the user needs to specify a?

Reply:
Done